src/main/resources/db/changesets/20181203_studytable.yaml

```diff
@@ -8,45 +8,62 @@ databaseChangeLog:
             columns:
               - column:
                   name: id
-                  type: UUID
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
                   constraints:
                     primaryKey: true
                     nullable: false
               - column:
                   name: name
-                  type: varchar(1000)
+                  type: varchar(1024)
                   constraints:
                     nullable: false
+              - column:
+                  name: description
+                  type: varchar(2048)
```

Comment:
Does the 'varchar' here support UTF-8 characters? Or does it need to be 'nvarchar'?
Take this as a general question for all varchar columns.

Reply:
Yes, I was able to create a study called ☃